### PR TITLE
cleanup transaction mapper

### PR DESCRIPTION
**Pull request by dhrubabasu**

### CODE REVIEW

**Comment by learyce** [comment]
Looks like an improvement, but can we add a few tests to the mapper file to make sure we don't accidentally cause a regression

**Comment by dhrubabasu** [comment]
> Looks like real improvement, but can we add a few tests to the mapper file to make sure we don't accidentally cause a regression

done!

**Review by patrick-ogrady — APPROVED** [review]
LGTM. Thanks for adding tests.

File: mapper/transaction.go [hunk 1]
```diff
@@ -439,133 +442,118 @@ func traceOps(trace []*clientTypes.FlatCall, startIndex int) []*types.Operation
 }
 
 func erc20Ops(transferLog *ethtypes.Log, currency *clientTypes.ContractCurrency, opsLen int64) []*types.Operation {
-	ops := []*types.Operation{}
-
 	contractAddress := transferLog.Address
-	addressFrom := transferLog.Topics[1]
-	addressTo := transferLog.Topics[2]
+	fromAddress := common.BytesToAddress(transferLog.Topics[1].Bytes())
```
Comment (patrick-ogrady) [inline mapper/transaction.go:446]: Very nuanced behavior on this guy: https://github.com/ethereum/go-ethereum/blob/830231c1c40f45f32cbf4ab69954b87a79313f44/common/types.go#L203-L204

**Review by learyce — APPROVED** [review]

**Pull request merged**